hdlparse/vhdl_parser.py

```diff
@@ -32,7 +32,8 @@
     (r'subtype\s+(\w+)\s+is\s+(\w+)', 'subtype'),
     (r'constant\s+(\w+)\s+:\s+(\w+)', 'constant'),
     (r'type\s+(\w+)\s*is', 'type', 'type_decl'),
-    (r'end\s+package', None, '#pop'),
+    (r'end\s+\w+\s*;', None, '#pop'),
+    (r'end\s+package\s+\w+\s*;', None, '#pop'),
```

Comment:
This line has a slightly different matching than the one above. It is probably not needed.

Reply:
I'll change that as it is for entity in #14 based on your comment.